packer/oem-build.json

```diff
@@ -10,6 +10,7 @@
     "git_branch": "master",
 
     "headless": "true",
+    "audio": "pulse",
```

Comment:
I am curious if we should just change the build-on-Linux and build-on-Windows scripts to just run packer and specify a value for audio. Those scripts are not used at all (as far as I am aware).

We could then just use the `null` audio driver here to ensure that the packer build never fails (even if audio doesn't work in the resulting VM). The relevant script would pass the right argument.

Reply:
I'm not excited about introducing a layer of indirection for something so minor. The scripts should probably be archived at this point, as the output has diverged so much from Packer. As far as I know, I'm the only one who's ever built this on Windows, and I can live with a command line flag. The script would also introduce another layer when trying to override other Packer options.

Reply:
Sounds good